sbi/inference/posteriors/score_posterior.py

```diff
@@ -222,11 +239,10 @@ def _sample_via_diffusion(
             )
         samples = torch.cat(samples, dim=0)[:num_samples]
 
-        return samples.reshape(sample_shape + self.score_estimator.input_shape)
+        return samples
 
     def sample_via_zuko(
```

Comment:
what do you think of renaming this to `sample_via_ode` to better reflect the method, not the package and to match the analog `sample_via_diffusion`?